scene/gui/rich_text_label.cpp

```diff
@@ -1719,6 +1884,8 @@ Error RichTextLabel::append_bbcode(const String &p_bbcode) {
 	bool in_bold = false;
 	bool in_italics = false;
 
+	set_process(false);
```

Comment:
This should likely use `set_internal_process()` so that it's independent of user scripts using `set_process` for their own logic.

Reply:
This has been resolved, but like the other one, I need to make sure it's fully compatible with my test godot project. I'll resolve this issue when I feel confident that there are no regressions.